docs/conf.py

```diff
@@ -46,7 +46,7 @@
 master_doc = 'contents'
 
 # General information about the project.
-project = 'Scylla Documentation'
+project = 'Scylla Dev'
```

Comment:
What happens when we branch? Perhaps this should read SCYLLA_VERSION_GEN.

Reply:
the release is presented in the right drop down, for example
https://scylla.docs.scylladb.com/branch-4.6/index.html
![image](https://user-images.githubusercontent.com/170200/164982165-02c3a6b0-08db-48ce-9ba7-4184e9bbb827.png)